ui/app/mirrors/create/cdc/tablemapping.tsx

```diff
@@ -103,7 +103,7 @@ export default function TablePicker({
           />
         </div>
       </div>
-      <div style={{ maxHeight: '70vh', overflow: 'scroll' }}>
```

Comment:
note: the maxHeight was blocking the UI list from being fully visible, so removed it here.